cli/src/main/java/de/jplag/cli/CliOptions.java

```diff
@@ -124,6 +127,18 @@ public static class Merging {
 
     }
 
+    public static class Csv {
+        @Option(names = {"--csv-print"}, description = "If true, the comparisons will pre printed in a csv%n")
+        public boolean print;
+
+        @Option(names = {
+                "--csv-anonymize"}, description = "If true, the csv will contain anonymized data and a second csv will contain the actual names.%n")
+        public boolean anonymize;
+
+        @Option(names = {"--csv-file-name"}, description = "Overrides the base name of the csv file. Do not include .csv in here.%n")
+        public String fileName = "resultCsv";
+    }
```

Comment:
Is this the path or the name? Only the name, right? Since the user cannot change the path, we could also think about a more simple solution in the CLI. For example, only one CSV flag. if this is enabled, both the anonymized and the unanonymized versions are written into two files. This flag can be part of the advanced group. This would keep the list of CLI options more concise and more readable.